pom.xml

```diff
@@ -41,16 +41,19 @@
         <dependency>
             <groupId>org.scalatest</groupId>
             <artifactId>scalatest_2.11</artifactId>
+            <version>3.0.0</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.scalacheck</groupId>
             <artifactId>scalacheck_2.11</artifactId>
+            <version>1.13.2</version>
```

Comment:
these version increases allow us to use the newest features of these libraries. ScalaCheck, for example, allows for better `Try` support. ScalaTest gives us new operators to verify the contents of sequences and ScalaMock-ScalaTest-Support needed to be updated accordingly to keep working properly.